docs/featurebasedmolecularnetworking-with-ms-dial.md

```diff
@@ -130,13 +136,13 @@ Go to Menu: Export / Alignment result
 
 ![img](img/ms-dial/export.png)
 
-Make sur the “GNPS export” box is checked, and set the spectra type to “centroid”.
+Make sure the “GNPS export” box is checked, and set the spectra type to “centroid”.
 
 Set the directory and file name, then click “Export”. As a result, you will acquire a MS/MS spectral summary and the feature quantification table.
 
 ![img](img/ms-dial/export_2.png)
 
-[Note] The GNPS export option of MS-DIAL will create a feature table based on MS1 peak height. If you want a feature table based on peak area, check the “Raw data matrix (Area)”.
+[Note] The GNPS export option of MS-DIAL will create a feature table based on MS1 peak height. If you want a feature table based on peak area, check the “Raw data matrix (Area)”.  The resulting file has a different structure from TXT file given by GNPS export, so you should reformat the area matrix file.
```

Comment:
Hey @KyobinKang, is it something that you observe only when MSe data is processed ? Or DDA as well ?

Reply:
Does not matter whether it's DDA or DIA. The GNPS export module of MS-DIAL gives a height-based table as a default. Maybe we should suggest a revision on this to Hiroshi?

Reply:
Indeed, I think that would be nice to have such an option. Lets reach out to Hiroshi to see if that is possible.